config/__init__.py

```diff
@@ -85,6 +85,8 @@ def split_args(sys_cmd: List):
     _use_ssl = '-s' if '-s' in pytest_args else '--use_ssl' if '--use_ssl' in pytest_args else None
     use_ssl = pytest_args[pytest_args.index(_use_ssl) + 1] if _use_ssl else True
     os.environ["USE_SSL"] = str(use_ssl)
+    _csm_checks = '-csm' if '-csm' in pytest_args else '--csm_checks' if '--csm_checks' in pytest_args else None
```

Comment:
Convert this into normal if statement. This line looks unreadable and extending beyond 100 chars.

Reply:
updated